<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>696</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>602</width>
    <height>696</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>602</width>
    <height>696</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color:#E0B9E7;
 background-position: center;
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DESCRIPTION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>620</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 50px;
background:#c68ad1;
opacity: 0.6;
}
QPushButton:hover {background-color: #a56baf; }</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>411</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
